util/src/org/commcare/util/mocks/MockDataUtils.java

```diff
@@ -141,8 +141,7 @@ public static FormInstance loadFixture(MockUserDataSandbox sandbox,
     }
 
     /**
-     * A quick way to request an evaluation context with an abstract instance available.
-     *
+     * Create an evaluation context with an abstract instance available.
      */
     public static EvaluationContext getInstanceContexts(MockUserDataSandbox sandbox, String instanceId, String instanceRef){
         InstanceInitializationFactory iif = new CommCareInstanceInitializer(sandbox);
```

Comment:
This should get generalized to call the mehtod below after creating a hashtable with its arguments, I think